NthKey/Model/Destination.swift

```diff
@@ -14,18 +14,13 @@ struct Destination : Identifiable, Equatable {
     public let id: String
     public let isChange: Bool
     
-    init(output: PSBTOutput, inputs: [PSBTInput]) {
-        // TODO: use wallet manager
-        let threshold = UInt(UserDefaults.threshold)
+    init(output: PSBTOutput, inputs: [PSBTInput], threshold: UInt, cosignerKeys: [HDKey]) {
         precondition(threshold > 0)
 
         self.description = String(output.txOutput.amount) + " sats" + ": " + output.txOutput.address!.description
         self.id = output.id
-        let (us, cosigners) = Signer.getSigners()
-        let cosignerKeys = cosigners.map { (signer) -> HDKey in
-            signer.hdKey
-        }
+        let us = Signer.getOurselfSigner()
+
         self.isChange = output.isChange(signer: us.hdKey, inputs: inputs, cosigners: cosignerKeys, threshold: threshold)
```

Comment:
Maybe `cosignerKeys` is not passed in correctly here, causing the change detection to break?

Reply:
@Sjors Actually, HDKeys did initialization use only description, without fingerprint (fixed in f880dcc). QR code was just a placeholder, after last fixes everything should work as far as my test shows same flow between `master` and current branch